homeassistant/components/caldav/calendar.py

```diff
@@ -142,13 +164,15 @@ async def async_get_events(self, hass, start_date, end_date):
     def update(self):
         """Update event data."""
         self.data.update()
-        self._event = self.data.event
+        event = copy.deepcopy(self.data.event)
```

Comment:
This appears to be reverting https://github.com/home-assistant/core/pull/72922 -- is that intentional? My impression is that this data class couldn't do a deepcopy.